packages/wp-now/src/start-server.ts

```diff
@@ -33,6 +33,16 @@ const requestBodyToString = async (req) =>
 		});
 	});
 
+function addTrailingSlash(path) {
+	return (req, res, next) => {
+		if (req.url === path) {
```

Comment:
Will that work if the URL is `/wp-admin?a=b`?

Reply:
I tested and it does not.

Maybe `addTrailingSlash` should be a standalone function with tests, and then we should have some other function for this middleware.

Reply:
- Now it supports get parameters `/wp-admin?a=b` 
- I moved the function to a new file, and included tests.